apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/conf/Config.java

```diff
@@ -164,4 +167,16 @@
             public static boolean TRACE_DSL = false;
         }
     }
+
+    public static class Customize {
```

Comment:
Customize plugin config should be an inner class of Plugin. Take `Plugin#Elasticsearch` as an example

Reply:
Agree. It looks better.